gradle/scripts/dependencyDefinitions.gradle

```diff
@@ -49,8 +49,9 @@ ext.externalDependency = [
     "commonsPool": "org.apache.commons:commons-pool2:2.4.2",
     "datanucleusCore": "org.datanucleus:datanucleus-core:3.2.10",
     "datanucleusRdbms": "org.datanucleus:datanucleus-rdbms:3.2.9",
+    "errorProne": "com.google.errorprone:error_prone_annotations:2.0.15",
```

Comment:
did I miss it?  I would have expected to see a new/adjusted import in this PR to bring in (e.g. annotations) from this new package.

Reply:
Only needed it in `gobblin-utility/build.gradle`, not sure if I should've added it to every build.gradle class?

Reply:
I saw that, but wouldn't there also be a new `import` in java code?  ...or are classes already imported under a package that stays fixed, but the package now comes from a separate jar than it used to?

overall, I'm still unclear (from reading only the code under PR) why this dep is needed?

Reply:
Errorprone is needed for some annotations such as `@IgnoreReturnValue` which is used in Guava transitively, but they removed the direct dependency on it so we have to include it manually.

It's discussed in this Github thread where the fix is to include errorProne as a compile dependency, otherwise we will have to bump to a future version of Guava https://github.com/google/guava/issues/2837

Reply:
yes, I see... makes sense there would be no `import`s needed in the case of one of our direct deps having left out their own dependency so it falls instead upon us to specify for them.  clearly preferable when transitive dependency resolution automates the solution... but now that we know what's missing, not too hard for us to add manually.